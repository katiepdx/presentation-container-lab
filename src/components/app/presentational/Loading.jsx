// Loading screen component 
// Needs a snapshot test