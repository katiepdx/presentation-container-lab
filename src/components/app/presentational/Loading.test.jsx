import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Loading from './Loading';


describe('Loading component', () => {
  afterEach(() => cleanup());
  
  it('displays loading image', () => {
    const { asFragment } = render(<Loading />);
    expect(asFragment()).toMatchSnapshot();
  });
});