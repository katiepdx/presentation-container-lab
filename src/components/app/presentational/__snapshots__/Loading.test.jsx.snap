// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component displays loading image 1`] = `
<DocumentFragment>
  <img
    alt="grey bubble loading spinner"
    src="https://miro.medium.com/max/882/1*9EBHIOzhE1XfMYoKz1JcsQ.gif"
  />
</DocumentFragment>
`;
